code/datums/gamemodes/combat_patrol.dm

```diff
@@ -0,0 +1,189 @@
+/datum/game_mode/combat_patrol
+	name = "Combat patrol"
+	config_tag = "Combat patrol"
+	flags_round_type = MODE_LZ_SHUTTERS|MODE_TWO_HUMAN_FACTIONS|MODE_HUMAN_ONLY|MODE_SOM_OPFOR //MODE_NO_PERMANENT_WOUNDS is for nerds
+	flags_landmarks = MODE_LANDMARK_SPAWN_SPECIFIC_SHUTTLE_CONSOLE
+	shutters_drop_time = 5 MINUTES
+	flags_xeno_abilities = ABILITY_CRASH
+	respawn_time = 12 MINUTES
+	time_between_round = 0 HOURS
+	valid_job_types = list(
+		/datum/job/terragov/squad/engineer = 4,
+		/datum/job/terragov/squad/corpsman = 8,
+		/datum/job/terragov/squad/smartgunner = 4,
+		/datum/job/terragov/squad/leader = 4,
+		/datum/job/terragov/squad/standard = -1,
+		/datum/job/som/squad/leader = 4,
+		/datum/job/som/squad/veteran = 4,
+		/datum/job/som/squad/engineer = 4,
+		/datum/job/som/squad/medic = 8,
+		/datum/job/som/squad/standard = -1,
+	)
+	/// Timer used to calculate how long till round ends
+	var/game_timer
+	///The length of time until round ends.
+	var/max_game_time = 40 MINUTES
+
+/datum/game_mode/combat_patrol/post_setup()
+	. = ..()
+	for(var/area/area_to_lit AS in GLOB.sorted_areas)
+		switch(area_to_lit.ceiling)
+			if(CEILING_NONE to CEILING_GLASS)
+				area_to_lit.set_base_lighting(COLOR_WHITE, 200)
+			if(CEILING_METAL)
+				area_to_lit.set_base_lighting(COLOR_WHITE, 100)
+			if(CEILING_UNDERGROUND to CEILING_UNDERGROUND_METAL)
+				area_to_lit.set_base_lighting(COLOR_WHITE, 75)
+			if(CEILING_DEEP_UNDERGROUND to CEILING_DEEP_UNDERGROUND_METAL)
+				area_to_lit.set_base_lighting(COLOR_WHITE, 25)
+	GLOB.join_as_robot_allowed = FALSE
```

Comment:
no robot ? Why not btw ?

Reply:
Because they ignore gas. And pain. And fire. And don't crit.
Basically I think they have an actually notable advantage in combat patrol, although it turns out this doesn't actually work, as atleast some people were able to join as combat robots today.